packages/salesforcedx-sobjects-faux-generator/package.json

```diff
@@ -9,6 +9,7 @@
     "vscode": "^1.32.0"
   },
   "dependencies": {
+    "@salesforce/core": "2.0.1",
```

Comment:
Were you able to understand why it breaks those extensions?

Reply:
Yes, those problems are caused by importing more than one module that depends on `@salesforce/core` into the same package. I looked at refactoring all the code that depends on it into the `utils` module and it will be a significant refactor and testing effort.